Source/UI/Message/MessageList.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  MessageList.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 19/12/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol MessageList: InfiniteList {
+    associatedtype CachedCollection = [Message]
```

Comment:
I think maybe this line is the only necessary one, and all those below it can be deleted since they are inherited from InfiniteList?

Reply:
👌